lib/ansible/modules/packaging/os/bindep.py

```diff
@@ -0,0 +1,199 @@
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: bindep
+short_description: Get lists of missing distro packages to be installed
+author: "Monty Taylor (@mordred)"
+version_added: "2.4"
+description:
+    - Retreive a list of packages that are not already installed based on a
+      list of requirements in bindep format. For more information on bindep,
+      please see https://docs.openstack.org/infra/bindep/
+requirements:
+    - "python >= 2.7"
+    - "bindep >= 2.2.0"
+options:
+   path:
+     description:
+       - Path to a bindep.txt file or to a directory that contains either
+         a bindep.txt file or an other-requirements.txt file that should
+         be used as input. Mutually exclusive with requirements.
+     required: false
+     default: None
+   requirements:
+     description:
+       - A list of bindep requirements strings. Mutually exclusive with path.
+     required: false
+     default: None
+   profiles:
+     description:
+       - An explicit list of profiles to filter by
+     required: false
+     default: ['default']
+'''
+
+EXAMPLES = '''
+# Get the list of packages to install on a given host from a file
+- bindep:
+    path: /home/example/bindep.txt
+
+# Get the list of packages to install and install them
+- bindep:
+    path: /home/example/bindep.txt
+  register: ret
+- package:
+    name: '{{ item }}'
+    state: latest
+  with_items: ret.bindep_packages.missing
+
+# Get the list of packages to install on a given host from a list
+- bindep:
+    requirements:
+        - build-essential [platform:dpkg test]
+        - gcc [platform:rpm test]
+        - language-pack-en [platform:ubuntu]
+        - libffi-dev [platform:dpkg test]
+        - libffi-devel [platform:rpm test]
+
+# Get the list of packages to install on a given host for the test profile
+- bindep:
+    requirements:
+        - build-essential [platform:dpkg test]
+        - gcc [platform:rpm test]
+        - language-pack-en [platform:ubuntu]
+        - libffi-dev [platform:dpkg test]
+        - libffi-devel [platform:rpm test]
+    profiles:
+        - test
+'''
+
+
+RETURN = '''
+bindep_packages:
+    description: Dictionary containing information about the requirements.
```

Comment:
Please change the indentation to 2 spaces like in the `DOCUMENTATION` and the `EXAMPLES` blocks.